### PR TITLE
Change default NUPCOL to 12

### PR DESCRIPTION
**Pull request by totto82**
Flow needs a default value of NUPCOL larger then 3.

### CODE REVIEW

File: src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp [hunk 1]
```diff
@@ -127,7 +127,7 @@ namespace {
         global_whistctl_mode(this->m_timeMap, Well2::ProducerCMode::CMODE_UNDEFINED),
         m_actions(this->m_timeMap, std::make_shared<Action::Actions>()),
         rft_config(this->m_timeMap),
-        m_nupcol(this->m_timeMap, 3)
+        m_nupcol(this->m_timeMap, 12)
```
Comment (joakim-hove) [inline src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp]: This should IMHO use the default from the Json scheme:
```
m_nupcol(this->m_timeMap, ParserKeywords::NUPCOL::NUM_ITER::defaultValue)
```

File: src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp [hunk 2]
```diff
@@ -536,7 +536,13 @@ namespace {
     }
 
     void Schedule::handleNUPCOL( const DeckKeyword& keyword, size_t currentStep) {
-        const int nupcol = keyword.getRecord(0).getItem("NUM_ITER").get<int>(0);
+        int nupcol = keyword.getRecord(0).getItem("NUM_ITER").get<int>(0);
+        if (keyword.getRecord(0).getItem("NUM_ITER").defaultApplied(0)) {
```
Comment (joakim-hove) [inline src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp]: I really think the default value should be updated in the Json file; possibly you can add a `NUPCOL` keyword with default value 12 here: https://github.com/OPM/opm-common/tree/b8605532f4d66dcb9c33476940622c4b62213588/src/opm/parser/eclipse/share/keywords/900_OPM -  then that will override the Eclipse100 version.

Comment (totto82) [reply src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp]: I will change the default in the Json file and add a comment there + keep the note if NUPCOL is used with 1*.

**Comment by joakim-hove** [comment]
jenkiins build this please

**Comment by akva2** [comment]
a bit iiiiiger

**Comment by akva2** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
@totto82 - can this jenkins failure be explained?

**Comment by totto82** [comment]
Yes. The results are sensitive to NUPCOL. The current NUPCOL behavior is insufficient, but this is addressed in OPM/opm-simulators#2144. Both this PRs needs an update of data. Maybe they should make the update of data together and be merged together.

**Comment by atgeirr** [comment]
Looks like the review issues were resolved. I'll merge this with the opm-tests and opm-simulators PRs now,  as jenkins is green and the PRs reviewed.

**Pull request merged**